awx/main/analytics/certificate_manager.py

```
@@ -0,0 +1,439 @@
+                return False
+
+            uuid = consumer_info['uuid']
+
+            try:
+                # GET /consumers/{uuid} -- this IS the check-in
+                # Resets lastCheckin server-side (connection.py:1862)
+                response = requests.get(
+                    f"{self.candlepin_url}/consumers/{uuid}",
+                    cert=(self.identity.certpath, self.identity.keypath),
+                    headers={'Accept': 'application/json'},
+                    verify=False,
```

Ruff (0.15.1)

[error] Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)